module/dice/damage-roll.mjs

```diff
@@ -130,7 +130,12 @@ export default class DamageRoll extends Roll {
 
           // Powerful critical - maximize damage and reduce the multiplier by 1
           if ( this.options.powerfulCritical ) {
-            let bonus = term.number * term.faces;
+            const maxterm = new Die({
+              number: term.number,
+              faces: term.faces,
+              modifiers: term.modifiers
+            }).evaluate({ maximize: true });
```

Comment:
As mentioned in the issue there is a work around for v12 if we make the configureDamage async and add the following code
`const version = parseInt(game.version.substring(0, game.version.indexOf(".")));
if (version <= 12) await maxterm._evaluateModifiers();`

Reply:
You can use `game.release.generation` to make this easier. Also, current version is V12 so what are you protecting against with this?

Reply:
I'm not sure how feasible it is to make `configureDamage` async because it is called within the roll's constructor.